### PR TITLE
Added logout button

### PR DESCRIPTION
**Pull request by jeroendesloovere**

### CODE REVIEW

File: src/Backend/Modules/Instagram/Layout/Templates/Settings.html.twig [hunk 1]
```diff
@@ -1,5 +1,17 @@
 {% extends '/Layout/Templates/base.html.twig' %}
 
+{% block actionbar %}
+  <div class="btn-toolbar pull-right">
+    <div class="btn-group" role="group">
+      {% if showInstagramLogout %}
```
Comment (jessedobbelaere) [inline src/Backend/Modules/Instagram/Layout/Templates/Settings.html.twig]: PP: `{% if showInstagramLogout and not authenticate %}`

File: src/Backend/Modules/Instagram/Layout/Templates/Settings.html.twig [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+          {% if not authenticate %}
+          {{ macro.buttonIcon( geturl('Logout'), 'close', 'lbl.Logout'|trans|ucfirst) }}
```
Comment (jessedobbelaere) [inline src/Backend/Modules/Instagram/Layout/Templates/Settings.html.twig:7]: PP: Maybe indent this?

**Comment by jessedobbelaere** [comment]
thanks, doing the hacktoberfest challenge :D ? https://hacktoberfest.digitalocean.com/

**Comment by jeroendesloovere** [comment]
Yep, my tshirt is underway ;-P

**Pull request closed without merging**